docs/source/index.rst

```diff
@@ -67,6 +67,15 @@
 
   Deploy Aerie for individual users, small and large missions.
 
+.. topic-box::
+  :title: External Datasets
+  :link: external-datasets
+  :icon: scylla-icon scylla-icon--resource-center
+  :class: large-4
```

Comment:
🔵 Any reason this one should be large-4 while the others are large-6?

![Screen Shot 2022-10-28 at 8 16 03 AM](https://user-images.githubusercontent.com/1189602/198672786-7e99b882-c75f-4908-90ad-f7da76b51cef.png)

Reply:
whoops I was experimenting with shrinking all the boxes, forgot to undo it.

Reply:
done